packages/storybook/stories/ProgressButton.stories.jsx

```diff
@@ -1,56 +0,0 @@
-import React, { Fragment } from 'react';
-import ProgressButton from '../../react-components/src/components/ProgressButton/ProgressButton';
-import { StoryDocs } from './wc-helpers';
-
-export default {
-  title: 'Components/Buttons/ProgressButton',
```

Comment:
Not that I'm complaining, but why are we deleting this story but leaving others as "Deprecated"?

Reply:
I decided to go ahead and delete it because we really don't want people using it and I don't feel like we need to maintain the documentation for it the way we would with a React component.

Reply:
It is a [React component](https://github.com/department-of-veterans-affairs/component-library/blob/9b58a8c0216c9215dd8b5b31fb3d90dea9f29227/packages/react-components/src/components/ProgressButton/ProgressButton.jsx) though.

![Report of `ProgressButton` component usage in vets-website. It is used 12 times total - 10 times in platform-ish code, once in the "covid-vaccine-expansion" application, and once in the "Coronavirus Research - Volunteer" application.](https://user-images.githubusercontent.com/2008881/187485071-b39c4a92-cb7c-4da3-9e6e-c04ffad25c2d.png)